runelite-client/src/main/java/net/runelite/client/plugins/hamstoreroom/HamOverlay.java

```diff
@@ -61,27 +56,22 @@ public Dimension render(Graphics2D graphics)
 		{
 			return null;
 		}
-		drawSteel = false;
-		drawIron = false;
-		drawSilver = false;
-		drawBronze = false;
-
 
 		if (hamPlugin.steelObject.getCanvasTilePoly() != null)
```

Comment:
Access the data in the plugin with `hamPlugin.getSteelObject()` for example, which will use the Getters that Lombok generated